src/main/java/com/googleinterns/zoomtube/data/TranscriptLine.java

```diff
@@ -26,28 +27,31 @@
   public static final String PROP_START = "start";
   public static final String PROP_DURATION = "duration";
   public static final String PROP_CONTENT = "content";
+  public static final String PROP_END = "end";
```

Comment:
Are these needed to be public? If they are used in two classes you can make a util class that just has TranscriptLineEntity field constants

Reply:
👍